styx-client/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java

```diff
@@ -69,8 +78,9 @@
   private static final String CLIENT_HOST = API_URL.scheme() + "://" + API_URL.host() ;
 
   @Before
-  public void setUp() throws Exception {
+  public void setUp() throws Throwable {
```

Comment:
Why the change here?

Reply:
Remnant from earlier change that got removed. Will revert.